src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -94,10 +94,10 @@ internal static TypeSymbol CopyTypeCustomModifiers(TypeSymbol sourceType, TypeSy
             // that the author did not write and did not validate.
             var flagsBuilder = ArrayBuilder<byte>.GetInstance();
             destinationType.AddNullableTransforms(flagsBuilder);
-            int position = 0;
-            int length = flagsBuilder.Count;
-            bool transformResult = resultType.ApplyNullableTransforms(defaultTransformFlag: 0, flagsBuilder.ToImmutableAndFree(), ref position, out resultType);
-            Debug.Assert(transformResult && position == length);
+            var stream = NullableTransformStream.Create(defaultTransform: 0, flagsBuilder.ToImmutableAndFree());
```

Comment:
>defaultTransform: 0,  [](start = 56, length = 21)

It looks like this parameter is no longer required and can be dropped for clarity.